app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -518,8 +519,14 @@ private String prettyUploadedDate(Media media) {
         if (date == null || date.toString() == null || date.toString().isEmpty()) {
             return "Uploaded date not available";
         }
-        SimpleDateFormat formatter = new SimpleDateFormat("yyyy MMM dd", Locale.getDefault());
-        return formatter.format(date);
+        String formatter;
```

Comment:
Add this function in `DateUtils` and make it generic so that other places can use the same thing.

Reply:
ok I'll add that